package.json

```diff
@@ -26,7 +26,8 @@
     "cordova:prepare": "cordova prepare",
     "cordova:build": "cordova build",
     "cordova:build:iso": "cordova build ios",
-    "cordova:build:android": "cordova build android"
+    "cordova:build:android": "cordova build android",
+    "mm": "./node_modules/.bin/mm"
```

Comment:
okay, for example I din't install webpack globally
and npm script **build** works fine without ./node_modules/.bin/webpack